common/src/WooCommerce.php

```diff
@@ -148,6 +149,10 @@ private function sendInvite($order_id) {
 
         // send invite
         $api = new API($api_domain, $shop_id, $api_key);
+        if ($this->plugin->getOption('invite') == 3 && !$api->hasConsent($data)) {
```

Comment:
Did you test what happens in case of an exception or bad request data, e.g. empty `order_number`, wrong or empty credentials